include/mbedtls/ssl.h

```diff
@@ -5224,6 +5224,11 @@ int mbedtls_ssl_close_notify(mbedtls_ssl_context *ssl);
  *                   same warnings apply to any use of the
  *                   early_exporter_master_secret.
  *
+ *                 Mbed TLS does not implement one of the anti-replay defenses
+ *                 defined in section 8 of the TLS 1.3 specification:
+ *                 single-ticket use or ClientHello recording within a given
```

Comment:
I think you mean single-use ticket ;) Also, I think it should say "does not implement _any_ of". (Otherwise, could be interpreted as "there's one of them that we're not implementing".)

Also, I'd make this a separate `\warning`: the previous one can be ignored by someone who's already super familiar with the spec, this one can't. So IMO they should be more clearly separated.